patch-package
--- a/node_modules/gift/lib/commit.js
+++ b/node_modules/gift/lib/commit.js
@@ -142,7 +142,7 @@
 
     Commit.actor = function(line) {
       var actor, epoch, m, ref1;
-      ref1 = /^.+? (.*) (\d+) .*$/.exec(line), m = ref1[0], actor = ref1[1], epoch = ref1[2];
+      //ref1 = /^.+? (.*) (\d+) .*$/.exec(line), m = ref1[0], actor = ref1[1], epoch = ref1[2];
       return [Actor.from_string(actor), new Date(1000 * +epoch)];
     };
 
